force-app/main/default/lwc/esa_LC_displayDECInformation/esa_LC_displayDECInformation.js

```diff
@@ -0,0 +1,24 @@
+import { LightningElement, api, wire } from 'lwc';
+import { getRecord, getFieldValue } from 'lightning/uiRecordApi';
+import ACCOUNT_NAME from '@salesforce/schema/Case.DEC__r.Name';
+import ACCOUNT_DEC from '@salesforce/schema/Case.DEC__r.DEC__c';
+//import labelName from '@salesforce/label/label-reference';
+
+const fields = [ACCOUNT_NAME,ACCOUNT_DEC];
+
+export default class Esa_LC_displayDECInformation extends LightningElement {
+    @api recordId;
+
+    @wire(getRecord, { recordId: '$recordId', fields })
+    case;
```

Comment:
@deepakmulamalla: Better to use other keywords like caseObjectRecord instead of standard object literals.